db/clean.sql

```diff
@@ -4,12 +4,13 @@ update donations set email = 'webmaster@mushroomobserver.org';
 update donations set who = 'anonymous' where anonymous = true;
 
 update herbaria set email = 'webmaster@mushroomobserver.org';
+update collection_numbers set name = 'xxx';
```

Comment:
Is this necessary? Isn't publicly visible? See partial screenshot from ttps://mushroomobserver.org/collection_number/show_collection_number/7939
![Screen Shot 2020-11-01 at 15 43 09](https://user-images.githubusercontent.com/45460/97818764-34e4a980-1c59-11eb-9d00-d3248041b3e1.jpg)

Reply:
I agree with Joe.  Is there a use-case you have in mind, Nathan?

Reply:
I spotted some email addresses there, but I guess it's not critical that it be emptied.